### ISSUE TITLE
Add standards --help and --version flags

### ISSUE THREAD

**Issue by oz** [francois2metz/node-intervals#2]
The current behaviour of:

```
$ intervals --help
Add 8 non billable hours for 2011-04-06
^C
```

or...

```
$ intervals --version
Add 8 non billable hours for 2011-04-06
^C
```

is a little bit confusing for the seasoned UNIX user. It's rather an enhancement than an issue, but github's bug-tracking system does not distinguish between the two so far. :)

**Comment by francois2metz** [comment]
I'm working on it.

**Comment by francois2metz** [comment]
Fixed in 78ed94d1b69c6be28e4e27952b0cadcba4a4111d.
